src/utils/azure.ts

```diff
@@ -10,6 +10,7 @@ import { getResourceGroupFromId, IStorageAccountWizardContext, parseAzureResourc
 import { AzureWizard, AzureWizardExecuteStep, IActionContext, IAzureQuickPickItem, ISubscriptionContext } from '@microsoft/vscode-azext-utils';
```

Comment:
I think that they are covered in #3390, but all the `@azure` imports above should be `import type`.